Signal/src/ViewControllers/ConversationView/ConversationStyle.swift

```diff
@@ -48,8 +48,8 @@ public class ConversationStyle: NSObject {
         }
     }
 
-    @objc public let contentMarginTop: CGFloat = 10
-    @objc public let contentMarginBottom: CGFloat = 10
+    @objc public let contentMarginTop: CGFloat = 24
+    @objc public let contentMarginBottom: CGFloat = 24
```

Comment:
These changes aren't complete or final, but they reflect conversations with Myles and are closer to where we'll land.